cogs/welcome/welcome.py

```diff
@@ -167,6 +143,49 @@ async def sendWelcomeMessage(self, newUser, test=False):
                         newUser.id,
                     )
 
+    async def sendLogUserDescription(self, user: discord.User):
+        """Sends the user's tagged description to the log channel if it exists"""
+        currentGuild = user.guild
+        guildConfig = self.config.guild(currentGuild)
+
+        isLogJoinEnabled = await guildConfig.get_attr(KEY_LOG_JOIN_ENABLED)()
+        if not isLogJoinEnabled:
+            return
+
+        logChannel = await guildConfig.get_attr(KEY_LOG_JOIN_CHANNEL)()
+        logChannel: discord.TextChannel = discord.utils.get(
+            currentGuild.text_channels, id=logChannel
+        )
```

Comment:
Curious, does this mess with your type hinting if you re-assign the variable with a different type?

Reply:
Both the variables and the type hints themselves are all good, though looking at this, I think it is better to also have type hint for line 155. VSCode, for example, can still recognize `logChannel` being `discord.TextChannel` after line 158.

Reply:
Resolved by renaming it to `logChannelId` in line 155.

Reply:
Ah okay, cool thanks!